File: seed/csharp-sdk/trace/src/SeedApi.DynamicSnippets/Example35.cs
using SeedTrace;

namespace Usage;

public class Example35
{
    public async Task Do() {
        var client = new SeedTraceClient(
            token: "<token>",
            clientOptions: new ClientOptions {
                BaseUrl = "https://api.fern.com"
            }
        );

        await client.V2.V3.Problem.GetProblemsAsync();
    }

}

File: seed/swift-sdk/trace/Tests/Snippets/Example35.swift
import Foundation
import Trace

enum Example35 {
    static func snippet() async throws {
        let client = TraceClient(
            baseURL: "https://api.fern.com",
            token: "<token>"
        )

        _ = try await client.v2.v3.problem.getProblems()
    }
}
